pnpm-lock.yaml

```diff
@@ -125,7 +125,7 @@ importers:
       '@babel/preset-env': 7.16.11_@babel+core@7.17.5
       '@babel/preset-react': 7.16.7_@babel+core@7.17.5
       '@babel/preset-typescript': 7.16.7_@babel+core@7.17.5
-      '@senecacdot/eslint-config-telescope': link:tools/eslint
+      '@senecacdot/eslint-config-telescope': 1.0.0
```

Comment:
These ones here are causing the eslint fails. 
It should have been changed in #3220 
cc @DukeManh

Reply:
In #3200, I added the `no-linking` option to pnpm, which won't link any package. But we wanted to link eslint so I linked eslint manually with `pnpm link` command, `pnpm-lock` is updated, but when we install a new dependency which the pr does, pnpm sees the `no-linking` option and unlink it.

I realized this today and make a fix for it in #3228. We should get it in before Dianna lands this.